common/src/leap/soledad/common/couch.py

```diff
@@ -1474,14 +1506,32 @@ def ensure_database(self, dbname):
         databases. But, in case it ever does, by raising an exception here we
         have one more guarantee that no modified client will be able to
         enforce creation of a database when syncing.
+        This can only be done by using a shell script with proper validation
+        and permissions (create_cmd).
```

Comment:
I think the diff is confusing the previous text, as the previous text is talking on how this throws an exception. Tt might make sense to remove all the text.